modules/charon-core/src/main/java/org/wso2/charon3/core/exceptions/BadRequestException.java

```diff
@@ -23,6 +23,10 @@
  */
 public class BadRequestException extends AbstractCharonException {
 
+    public BadRequestException() {
+        this("invalid request");
```

Comment:
@Captain-P-Goldfish Can you please define this error message as a constant in `ResponseCodeConstants` class?

Reply:
done